### PR TITLE
All the basics

### PR DESCRIPTION
**Pull request by ZeProgFactory**
- Links
- Session list
- Images

more to come later ;-)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,173 @@
...
-Repository for material/slides of the XamExpertDay2021
+<img src="_Images_/XamExpertDay2021.png" width="800" />
+
+
+
+# Xamarin Expert Day 2021 (@XamExpertDay)
+
+All slides and relevant things from the XamExpertDay2021. The event was held virtually on October 1st  2021.
+
+https://expertday.forxamarin.com/
```
Comment (hot33331) [inline README.md]: URL is wrong and does not work. Please use https://www.xamexpertday.com

File: README.md [hunk 2]
```diff
@@ -1,2 +1,173 @@
...
+### Creating PDF files in mobile apps with PdfSharpCore and MigraDocCore
+**Luis Beltran**  
+PDFsharpCore and MigraDocCore are Open Source .NET libraries that can be used to easily create and processes PDF documents on the fly in our Xamarin applications.  
+
+Handling documents with paragraphs, tables, styles, images, etc. and rendering them into PDF is possible with these libraries for free even in commercial applications, so in this session we'll see what it takes to develop a mobile application that creates PDF documents in several scenarios.  
+
+
+
+
+### Do you know your users?
```
Comment (hot33331) [inline README.md]: Session got removed

File: README.md [hunk 3]
```diff
@@ -1,2 +1,173 @@
...
+
+
+### An introduction to modern desktop app development with .NET and MAUI  
+**Konrad Müller** - [@KonMue](https://twitter.com/konmue)  
+In this session, we will take a look at the desktop development updates MAUI will bring to us and take a peek at other modern ways to develop for desktop with .NET. Get ready for a session full of old and new friends like UWP, WPF, Mac Catalyst, Blazor, PWAs and more!   
+
+
+
+
+### Building desktop apps using .NET MAUI and Blazor
```
Comment (hot33331) [inline README.md]: Session got removed

**Review by hot33331 — CHANGES_REQUESTED** [review]
Update only makes sense after today, there might be more changes to the schedule. But thank you!

File: README.md (hunk 1 above)
Comment (ZeProgFactory) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (ZeProgFactory) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (ZeProgFactory) [reply README.md]: done

**Review by jfversluis — APPROVED** [review]

**Pull request merged**